Benchmarks/Basic/BenchmarkRunner+Basic.swift

```diff
@@ -24,9 +24,9 @@ let benchmarks = {
                                                                 .p99: .milliseconds(3),
                                                                 .p100: .milliseconds(1)]
 
-        thresholds = [BenchmarkMetric.wallClock: BenchmarkThresholds(absolute: absolute)]
+        thresholds = [.wallClock: .init(absolute: absolute)]
```

Comment:
```suggestion
        thresholds = [.wallClock: .absolute(absolute)]
```
What do you think about this syntax?